openmoc/options.py

```diff
@@ -169,15 +171,15 @@ def parseArguments(self):
                 num_azim += 'the number of azimuthal angles\n'
                 print(num_azim)
 
-                num_polar = '\t{: <35}'.format('-a, --num-polar=<6>')
+                num_polar = '\t{: <35}'.format('-r, --num-polar=<6>')
                 num_polar += 'the number of polar angles\n'
                 print(num_polar)
 
                 azim_spacing = '\t{: <35}'.format('-s, --azim-spacing=<0.1>')
                 azim_spacing += 'The azimuthal track spacing [cm]\n'
                 print(azim_spacing)
 
-                polar_spacing = '\t{: <35}'.format('-s, --polar-spacing=<0.1>')
+                polar_spacing = '\t{: <35}'.format('-l, --polar-spacing=<0.1>')
```

Comment:
0.1 is too tight for a default polar angle spacing. 1.5 is probably better, since convergence is around 0.75 for PWR systems as far I know

Reply:
Yes, 0.1 is too tight. But I prefer to set the default value 0.75 since sometimes the axial mesh size might be as small as 1cm.

Reply:
1 cm mesh size is very fine, these default options are not meant to be very fine. 
Can a user with 8GB ram run any 3D small case (pincell ? C5G7 ?) with 0.75cm ?

Reply:
There might be some test problems created by the users, and even in the BEAVRS benchmark, the lowest fuel rod pin cell is about 1cm. What I concern most is there might be completely no track crossing the mesh if the axial spacing is greater than the axial size of the short mesh. I am not sure what would happen in this situation. Code crashed, divergence, or smoothly running?

Reply:
In order for the tracks to form cycles, the polar angle and spaces are corrected and there will always be tracks at every axial level. However, a FSR could be missed, but in this case, the code returns an explicit error message when computing the FSR volumes, telling the user to use a finer quadrature or spacing.